### DISCUSSION TITLE
Changing the points (on line) fill onover.

### DISCUSSION THREAD

**Discussion by vaibhav135** [naver/billboard.js#3052] [Q&A]
Hi, 
I am trying to change the point fill on hover but am not able to. When I inspected the circle. It showed the fill property is set as important. 
for example, you can try to change the fill on this js-fiddle example https://jsfiddle.net/45tsgkdL/4/  but it won't work.
Not sure how I can do that.

**Comment by netil** [comment]
Just override the `.bb-circle` rule with desired color, not by inlined approach.
- https://jsfiddle.net/netil/6fbstxL7/
